### ISSUE TITLE
The commit author in the Wiki repository is appearing as "undefined".

### ISSUE THREAD

**Issue by alexfdealmeida** [rfennell/AzurePipelines#1849]
### Azure DevOps Extensions

WIKI Updater

### Platform

Azure DevOps Services

### Extension Version

3.21.11

### Describe the bug

The username set in the "gitname" property is not being correctly set for the ".git/config" file. In other words, after running the "git config --local -l" command in the Wiki repository, it appears as "user.name=undefined". Regarding the "gitemail" property, it is working correctly.

### Repo Steps

Set up and run a pipeline in Azure DevOps.

```
      - task: WikiUpdaterTask@3
        name: publishProjectWiki
        displayName: 'Publish Project Wiki'
        condition: and(eq(canceled(), false), succeeded())
        inputs:
          repo: 'dev.azure.com/grupo-siagri/simer/_git/$(repoWikiName)'
          branch: '$(wikiBranchName)'
          localpath: '$(repoWikiPath)'
          filename: '$(releaseNotesWikiPath)/$(releaseNotesWikiName)'
          replaceFile: true
          dataIsFile: true
          sourceFile: '$(Build.SourcesDirectory)/ReleaseNotes.md'
          updateOrderFile: true
          prependEntryToOrderFile: true
          orderFilePath: '$(releaseNotesWikiPath)'
          message: 'Branch: $(wikiBranchName); Release Notes updated by Azure DevOps Pipeline: $(releaseNotesWikiNameCommitMsg)'
          gitname: '$(gitUserName)'
          gitemail: '$(gitUserEmail)'
          useAgentToken: true
```

### Expected Behavior

After committing to the Wiki repository, it should be defined with the author set in the 'gitname' property.

### Logging Information

Pipeline:
![image](https://github.com/rfennell/AzurePipelines/assets/308615/97497081-208b-4596-a125-5ebdc3dbb42c)

.git/config:
![image](https://github.com/rfennell/AzurePipelines/assets/308615/7600bf85-74d4-4d10-883d-dbf73e21a1ac)

commit:
![image](https://github.com/rfennell/AzurePipelines/assets/308615/a3ab03d7-45ba-4267-a1f2-29298ee04c29)

**Comment by rfennell** [comment]
I have a repo for that, looks like it has been an bug since day 1, it is a naming clash between a local variable and a parameter

I will get a fix out ASAP

**Comment by rfennell** [comment]
Being shipped as 3.22.3
